packages/manager/modules/models/BillingService/BillingService.class.js

```diff
@@ -329,6 +329,10 @@ export default class BillingService {
     );
   }
 
+  isSuspendedHostingWeb() {
+    return this.isHostingWeb && this.isResiliated();
```

Comment:
I wonder if when you skip the retention period and delete it for good, if this will be still considered had "suspended hosting web"